scaffold/github/workflows/Security.yml

```diff
@@ -0,0 +1,86 @@
+name: "Security"
```

Comment:
Forgive me if I’ve just missed the conversation, but last we discussed we thought we’d include this in the static check job to avoid the additional CI spin up costs of a separate ddev runner.

Reply:
Oh I misread this, we are at least combining two jobs together. 👍